Source/WebCore/html/HTMLOptionElement.cpp

```diff
@@ -93,6 +93,12 @@ bool HTMLOptionElement::isFocusable() const
 {
     if (!supportsFocus())
         return false;
+    
+    RefPtr select = ownerSelectElement();
+    if (select && select->usesMenuList())
+        return false;
+    return HTMLElement::isFocusable();
+
     // Option elements do not have a renderer.
     auto* style = const_cast<HTMLOptionElement&>(*this).computedStyle();
     return style && style->display() != DisplayType::None;
```

Comment:
The code below is now no longer executed.

Reply:
Should I move this to separate function: `HTMLOptGroup::isNotFocusableInMenuList` and rather than using `(select && select->)`, just use `(select->usesMenuList)`?

Reply:
Also in return, rather than returning 'HTMLElement::isFocusable', I return 'HTMLOptionElement::isFocusable'?